mavis/validate/base.py

```diff
@@ -862,6 +862,9 @@ def filter_if_true(read):
             try:
                 mates = self.bam_cache.get_mate(flanking_read, allow_file_access=False)
                 for mate in mates:
+                    if mate.is_unmapped:
+                        log('ignoring unmapped mate', mate.query_name)
```

Comment:
I thought we were going to log these for debug only? 
```suggestion
                        log('ignoring unmapped mate', mate.query_name, level=logging.DEBUG)
```